config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  resources :applications do
+    member do
+      patch :add
+    end
```

Comment:
One of the restrictions of the project is we can't use resources for routes. They have to be hand-rolled ie. get "/applications", to: "applications#index"

Reply:
Good catch! We should remove this. This was a left over when I was troubleshooting the button_to patch method.